server/app.js

```diff
@@ -8,6 +8,7 @@ const connectDB = require("./db");
 const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
+require("dotenv").config({ path: './sample.env' });
```

Comment:
The `sample.env` is important to other collaborators to have. This is just what the `.env` should have. By ignoring the sample.env onboarding future collaborators would be more complicated. I would revert this and just create a `.env` on your environment.

Reply:
Ah, okay thanks for the feedback Ethan! I just implemented your changes.